packages/help-center/src/components/back-button.tsx

```diff
@@ -24,16 +24,12 @@ export const BackButton = () => {
 	return (
 		<span className="back-button__help-center">
```

Comment:
Approved. But it would be ace to remove this extraneous [span](https://github.com/Automattic/wp-calypso/pull/97974/files#diff-89054efc3dd793e875318f84ebe5fecdffcc263915c285dd9f804966eecc8336R25).